### PR TITLE
Migrate businesses page

### PR DESCRIPTION
**Pull request by AndyClifford**
**Ticket:** https://www.notion.so/centrapay/Migrate-API-Documentation-c6d64de29a2245358a5e9b7eed5edbcc?pvs=4

### CODE REVIEW

File: src/content/api/businesses.mdx [hunk 1]
```diff
@@ -0,0 +1,417 @@
+---
+title: Businesses
+description: Business model and related endpoints.
+draft: true
+nav:
+  path: API
+  order: 9
+---
+import Properties from '../../components/Properties.astro';
```
Comment (AndyClifford) [inline src/content/api/businesses.mdx:9]: I wonder if there is a way we don't need to import these components in every markdown file

File: src/content/api/businesses.mdx [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Business.
+  </Property>
+
+  <Property name="updatedBy" type="crn">
+    The User or API Key that updated the Business.
+  </Property>
+
+  <Property name="taxNumber" type="object">
+    The value-added tax configuration for the Business.
```
Comment (AndyClifford) [inline src/content/api/businesses.mdx:67]:
```suggestion
    The value-added tax configuration for the Business. See [Tax Number Model](#tax-number-model)
```

File: src/content/api/businesses.mdx (hunk 1 above)
Comment (MeganSteenkamp) [reply src/content/api/businesses.mdx:9]: @AndyClifford I've been wondering that too 🤔 
Couldn't see anything in the Astro docs about it.

File: src/content/api/businesses.mdx [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import CodeGroup from '../../components/CodeGroup.astro';
+import Badge from '../../components/Badge.astro';
+
+A Business represents a company registered with the New Zealand Companies Office. It is associated with a single [Account](https://docs.centrapay.com/api/accounts).
+
+## Business Model
```
Comment (MeganSteenkamp) [inline src/content/api/businesses.mdx:17]: Suggestion: We can just say Model seeing as we are on the businesses page

File: src/content/api/businesses.mdx [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Business.
+  </Property>
+
+  <Property name="updatedBy" type="crn">
+    The User or API Key that updated the Business.
+  </Property>
+
+  <Property name="taxNumber" type="object">
+    The value-added tax configuration for the Business. See [Tax Number Model](#tax-number-model)
```
Comment (MeganSteenkamp) [inline src/content/api/businesses.mdx:67]:
```suggestion
    The value-added tax configuration for the Business. See [Tax Number Model](#tax-number-model).
```

File: src/content/api/businesses.mdx [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+
+    <Property name="accountId" type="string">
+      The Centrapay accountId.
+    </Property>
+
+    <Property name="test" type="string">
+      The created Centrapay org account will have the test flag.
+    </Property>
+
+    <Property name="taxNumber" type="string">
```
Comment (MeganSteenkamp) [inline src/content/api/businesses.mdx:138]:
```suggestion
    <Property name="taxNumber" type="object">
```

**Comment by MeganSteenkamp** [comment]
@AndyClifford LGTM, will approve after that one data type is fixed 🤠

File: src/content/api/businesses.mdx [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+|          Reason           |                           Description                            |
+| :------------------------ | :--------------------------------------------------------------- |
+| duplicate                 | The business already exists in the Centrapay system.             |
+| data-quality-issues       | Data quality issues are preventing the business from onboarding. |
+| no-response               | No response has been received from the business.                 |
+| change-of-ownership       | The business has changed ownership.                              |
+| centrapay-discontinued    | The business is no longer using Centrapay.                       |
+| seasonal-business-closure | The business has closed temporarily.                             |
+
+## Tax Number Model
```
Comment (MeganSteenkamp) [inline src/content/api/businesses.mdx:100]: Suggestion: Put a hr `---` between each model like @alexBinns has done: https://github.com/centrapay/centrapay-docs/pull/818/files

**Comment by MeganSteenkamp** [comment]
@AndyClifford Suggestion: When we are referring to `Business` we should use a proper noun as it is an object in our system.

**Review by MeganSteenkamp — APPROVED** [review]

**Pull request merged**